backend/climateconnect_main/settings.py

```diff
@@ -212,3 +210,7 @@
         }
     }
 }
+
+# For celery we are using redis as a broker URL
```

Comment:
Neat. What was the rationale on choosing Redis (over another option like RabbitMQ) as our broker? Just more familiarity and we're already using it in the project?

```suggestion
# For Celery we use Redis as a broker URL
```

Reply:
Two reasons....

1) We are already using Redis for chat messaging. i.e.: For our integration of django-channels (https://channels.readthedocs.io/en/stable/). 

2) The usage we are thinking here we do not require a dedicated message broker like RabbitMQ. Also we have more familiarity with Redis than RabbitMQ.

I would like your input though. I have found that Celery with Redis works pretty well for most use cases.

Reply:
Awesome, thanks for the context. I've limited experience with both (I'm not familiar with how one or the other performs under scale for our use cases) so I'd err on the side of more familiarity.

Reply:
Yeah I think redis can handle more complex and performs well under scale. I have used redis heavily before. But if time comes it easy to change the from redis to rabbtimq.